.github/workflows/build-notebooks-TEMPLATE.yaml

```diff
@@ -636,7 +636,7 @@ jobs:
           # mount the image
           IMAGE_MOUNT_DIR=$(sudo "${PODMAN}" image mount "${{ steps.calculated_vars.outputs.OUTPUT_IMAGE }}")
           # run the check-payload scan
-          sudo --preserve-env=PATH go run github.com/openshift/check-payload scan local --path "${IMAGE_MOUNT_DIR}"
+          sudo --preserve-env=PATH go tool github.com/openshift/check-payload scan local --path "${IMAGE_MOUNT_DIR}"
```

Comment:
Are we sure of the usage `tool` here? 
I read that It does not work with external packages like `github.com/openshift/check-payload` as it works with only for built-in Go tools (e.g. compile, vet, cover, nm, etc.).